jest/setup.js

```diff
@@ -33,3 +33,17 @@ import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 Enzyme.configure({ adapter: new Adapter() });
+
+Object.defineProperty(window, 'matchMedia', {
+  writable: true,
+  value: jest.fn().mockImplementation(query => ({
+    matches: false,
+    media: query,
+    onchange: null,
+    addListener: jest.fn(), // deprecated
+    removeListener: jest.fn(), // deprecated
```

Comment:
should we use these, if they are deprecated?

Reply:
As it is just a mock this might help if some libraries do use theses functions. We ourselves do not use this.